llvm/test/tools/llvm-objcopy/ELF/Inputs/srec-elf-segments.yaml

```diff
@@ -0,0 +1,60 @@
+# Test copied from ihex-elf-segments.yaml:
```

Comment:
The relationship with the ihex test is largely irrelevant. I shouldn't need to look at a different test to understand the logic of why this input contains what it does. Instead, properly comment on what is significant about this input (and delete any parts of the input that aren't relevant to the test).